pyccel/stdlib/gFTL_functions/Map_extensions.inc

```diff
@@ -44,5 +44,22 @@ contains
 
   end function __IDENTITY(Map)_pop_with_default
 
+  subroutine __IDENTITY(Map)_popitem(my_map, key, value)
+    class(Map), intent(inout) :: my_map
+    __Key_declare_dummy__, intent(out) :: key
+    __T_declare_dummy__, intent(out) :: value
+
+    type (__map_iterator) :: iter
+    type (__map_iterator) :: new_iter
+
+    iter =  my_map%begin()
+
+    key = iter % first()
+    value = iter % second()
+
+    new_iter = my_map % erase( iter )
```

Comment:
If `new_iter` is just a local iterator, why do we need it?

Reply:
Removed: bd4535c08